src/components/DayPicker.jsx

```diff
@@ -142,10 +144,12 @@ export default class DayPicker extends React.Component {
       currentMonth: props.hidden ? moment() : props.initialVisibleMonth(),
       monthTransition: null,
       translationValue: 0,
+      scrollableMonthMultiple: 1,
```

Comment:
I'm adding new months in batches instead of one by one, and this variable tracks how many batches have been added. For example, if there are initially 12 scrollable months and the user clicks the next button, I'm adding another 12 months by incrementing this variable.